### ISSUE TITLE
make this a library afterall....

### ISSUE THREAD

**Issue by gregplaysguitar** [sonsandco/django-shoptools#3]
I feel like this code could benefit from being treated as a library, pip installable etc. That would mean we can't just hack it per project as we do now but we'd always have the option of copying the code into src and doing it there. Thoughts? @pbright

**Comment by gregplaysguitar** [comment]
Proposal: 
- library to be published on pypi as django-shoptools
- all code to be moved into the shoptools namespace, so INSTALLED_APPS would contain 'shoptools.cart', 'shoptools.checkout', etc 
- current shoptools app becomes shoptools.core (or a better name?)
- payment apps/wrappers live under shoptools.payment
- vouchers, wishlist become shoptools.contrib.vouchers, shoptools.contrib.wishlist

@pbright, @brendonjohn any thoughts? If not I'll probably action this pretty soon when I get some time

**Comment by pbright** [comment]
The Paypal app is not ready for public consumption. It works but it's far from feature complete and a bit hacky in places. Apart from that sounds fine.

**Comment by gregplaysguitar** [comment]
Yup, I was going to remove that, when/if it's prod-ready we can put it back in. The shipping app also needs some attention, see #12
